Makefile

```
@@ -1,85 +1,99 @@
-help:
-	@echo 'start                  - start the server'
-	@echo 'restart                - rebuild and start the server'
-	@echo 'fresh                  - perform a fresh build, install and start the server'
-	@echo 'createsuperuser        - create a new django superuser'
-	@echo 'makemigrations         - create migration files'
-	@echo 'migrate                - run django migrations'
-	@echo 'dumpdata               - dump current data stored into ./app/fixture.json'
-	@echo 'loaddata               - load fixtures from ./app/fixture.json into the database'
-	@echo 'collectstatic          - collect static files to a single location to be served in production'
-	@echo 'test                   - run test suite'
-	@echo 'format                 - format code and imports'
-	@echo 'check                  - check formatting, imports and linting'
-	@echo 'black                  - format code only'
-	@echo 'isort                  - format imports only'
-	@echo 'flake8                 - check code style'
-
-## Start the webserver with docker on http://localhost:8000
-start:
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: ## This help.
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+.PHONY: start
+start: ## Start the webserver with docker on http://localhost:8000
 	docker-compose up
 
-down:
+.PHONY: down
+down: ## Take down server
 	docker-compose down -v
 
-restart:
+.PHONY: restart
+restart: ## Rebuild and start the server
 	docker-compose build
-	docker-compose up
+	make start
 
-fresh:
+.PHONY: fresh
+fresh: ## Perform a fresh build, install and start the server
 	docker-compose build
 	make makemigrations
 	make migrate
 	make loaddata
-	docker-compose up
+	make start
+
 
-createsuperuser: ##@Docker Create a superuser
+.PHONY: createsuperuser
+createsuperuser: ## Create a new django superuser
 	docker-compose run --rm web python manage.py createsuperuser
 
-makemigrations: ##@Docker Set up migration files
+
+.PHONY: makemigrations
+makemigrations: ## Create migration files
 	docker-compose run --rm web python manage.py makemigrations
 
-migrate: ##@Docker Perform migrations to database
+.PHONY: migrate
+migrate: ## Run django migrations
 	docker-compose run --rm web python manage.py migrate
```

Hadde vært smud med `args` her slik at man for eks kan kjøre `make migrate args="groups 0006"` for å migrere tilbake
```suggestion
	docker-compose run --rm web python manage.py migrate ${args}
```